lib/import.rb

```
@@ -0,0 +1,26 @@
+require_relative 'imgur'
+require_relative 'database'
+require 'dotenv'
+
+Dotenv.load
+db = Database.new
+
+['kitten', 'puppy'].each do |animal|
+  Imgur.gallery_images(animal).each do |image|
+    begin
+      db.insert(
+        :animals,
+        {
+          imgur_id: image['id'],
+          link: image['link'],
+          type: animal,
+          created_at: Time.now(),
+          updated_at: Time.now(),
+        })
+    rescue Sequel::UniqueConstraintViolation
```

Do not suppress exceptions.